go/vt/topo/vschema.go

```diff
@@ -82,10 +83,21 @@ func (ts *Server) EnsureVSchema(ctx context.Context, keyspace string) error {
 		}
 	}
 
-	err = ts.RebuildSrvVSchema(ctx, []string{} /* cells */)
```

Comment:
What if we change `EnsureVSchema` to accept a cells list to rebuild, and we have each tablet rebuild only its own cell? Would that still have caused the problem you had?

Reply:
Oh, I like that a lot more. I'll switch to that approach! 

That would've been fine. The problem was that the tablet in the init loop wasn't just rebuilding its own cell it was rebuilding all the other cells.